internal/oem/oem.go

```diff
@@ -244,6 +245,34 @@ alias gsutil="(docker images google/cloud-sdk || docker pull google/cloud-sdk) >
 		name:  "file",
 		fetch: file.FetchConfig,
 	})
+	configs.Register(Config{
+		name:  "sakuracloud",
+		fetch: sakuracloud.FetchConfig,
+		baseConfig: types.Config{
+			Systemd: types.Systemd{
+				Units: []types.SystemdUnit{
+					{
+						Enable: true,
+						Name:   "coreos-metadata-sshkeys@.service",
+					},
+					{
+						Enable: true,
+						Name:   "coreos-sakuracloud-network.service",
+						Contents: `[Unit]
```

Comment:
Typically, these units are defined in the OEM, so this could be moved to https://github.com/coreos/coreos-overlay/pull/2460/files. We then use the files section to [copy it from the OEM](https://github.com/coreos/ignition/blob/987205ea63f70982f4d7b8bf381063b07f3896ef/internal/oem/oem.go#L303) into the correct location. It's hacky, but we'll clean that up eventually.